ui/lib/core/addon/components/search-select.hbs

```diff
@@ -0,0 +1,102 @@
+<div
+  {{did-insert (perform this.fetchOptions)}}
+  {{did-update (perform this.fetchOptions) (or @options @models)}}
+  id={{@id}}
+  class={{concat "field search-select" (if (eq @displayInherit true) " display-inherit")}}
+  data-test-component="search-select"
+  ...attributes
+>
+  {{#if this.shouldUseFallback}}
+    {{component
+      @fallbackComponent
+      id=@id
+      label=@label
+      subText=@subText
+      onChange=@onChange
+      inputValue=@inputValue
+      helpText=@helpText
+      placeholder=@placeholder
+    }}
```

Comment:
I wonder if this would be better as a `yield` in the future? Maybe not if it's typically the same component like an input passed in but just a thought.

Reply:
Yeah - this is the block I was referring to in the dev channel last week. We only ever use `InputSearch` or `StringList` so I wondered about directly placing those here. I think yield would make sense in a figure iteration, especially if we move the management of selections to the parent. 

What do you think @zofskeez?

Reply:
I think it's good to leave as is so we don't blow out the scope.